{{cookiecutter.project_slug}}/README.md

```diff
@@ -10,6 +10,18 @@
 
 ## Features
 
+{% if cookiecutter.project_layout == "src" -%}
```

Comment:
it src or flat layout is not a feature, it is more like an organization.
so we don't need to have any information about that in the README file

maybe we could start a CONTRIBUTING.md file with a minimum information for now
and add this information as a note like (just an idea):

This project uses XYZ layout, that means that .. bla bla bla

Reply:
Fine.

Reply:
@xmnlab The file you indicate would be the one in this path: `{{cookiecutter.project_slug}}/docs/contributing.md`?

Reply:
@xmnlab If so, in which section could you add it?

Reply:
at the beginning of the document.  it is just a simple text explaining about the project layout.